### PR TITLE
[BugFix] Persist `LoadStatistic` in `EtlStatus`

### LINKED ISSUE #3555: Loss the JobDetails in the ShowLoad Stmt after fe restart
### Steps to reproduce the behavior (Required)
1. Execute the `show load\G` Stmt.
2. Restart Fe
3. Executor the `show load\G` again

### Expected behavior (Required)
Same with before
### Real behavior (Required)
image

### StarRocks version (Required)
 - You can get the StarRocks version by executing SQL `select current_version()`

### PR DESCRIPTION
**Pull request by padmejin**
## What type of PR is this：
- [x] bug
- [ ] feature
- [ ] enhancement
- [ ] refactor
- [ ] others

## Which issues of this PR fixes ：

Fixes #3555 

## Problem Summary(Required) ：

The main reason is that `LoadJob` did not persist `LoadStatistic` in the process of either replaying or snapshot. Unfortunately, there is no obvious way to add new metadata since we persist the `LoadJob` class in a hard-coded way. After due consideration, we serialized `LoadStatics` in JSON for the convenience of adding/deleting fields in the future and hide this JSON in a deprecated map in `EtlStatus`.

### CODE REVIEW

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:disappointed: **fail** : 15 / 60 (25.00%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/load/loadv2/SparkLoadJob.java|0|3|00.00%|[325, 328, 329]|
|:large_blue_circle:|com/starrocks/load/loadv2/LoadJob.java|0|4|00.00%|[219, 223, 227, 708]|
|:large_blue_circle:|com/starrocks/load/loadv2/BulkLoadJob.java|0|1|00.00%|[228]|
|:large_blue_circle:|com/starrocks/load/EtlStatus.java|15|52|28.85%|[103, 251, 252, 253, 327, 328, 329, 330, 331, 332, 335, 336, 339, 340, 341, 342, 343, 347, 348, 349, 350, 352, 353, 355, 359, 360, 361, 362, 364, 365, 366, 367, 368, 369, 370, 371, 372]|

File: fe/fe-core/src/main/java/com/starrocks/load/EtlStatus.java [hunk 1]
```diff
@@ -22,26 +22,48 @@
...
     private String trackingUrl;
-    private Map<String, String> stats;
+
+    /**
+     * This field is useless in runtime
+     * It only used for persisting LoadStatistic in consideration of compatibility
+     * It has only one k-v pair:
+     *   the key is LOAD_STATISTIC; the value is json string of loadStatistic
+     */
+    @Deprecated
```
Comment (gengjun-git) [inline fe/fe-core/src/main/java/com/starrocks/load/EtlStatus.java:62]: Why set to deprecated? Internal use is also considered use.

**Review by chaoyli — DISMISSED** [review]

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:disappointed: **fail** : 49 / 62 (79.03%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/load/loadv2/SparkLoadJob.java|0|3|00.00%|[325, 328, 329]|
|:large_blue_circle:|com/starrocks/load/loadv2/LoadJob.java|0|4|00.00%|[219, 223, 227, 708]|
|:large_blue_circle:|com/starrocks/load/loadv2/BulkLoadJob.java|0|1|00.00%|[228]|
|:large_blue_circle:|com/starrocks/load/EtlStatus.java|49|54|90.74%|[338, 339, 340, 341, 342]|

**Review by chaoyli — APPROVED** [review]

**Review by Astralidea — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
run starrocks_admit_test

**Comment by wanpengfei-git** [comment]
run starrocks_admit_test

**Comment by padmejin** [comment]
run all

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:disappointed: **fail** : 49 / 62 (79.03%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/load/loadv2/SparkLoadJob.java|0|3|00.00%|[325, 328, 329]|
|:large_blue_circle:|com/starrocks/load/loadv2/LoadJob.java|0|4|00.00%|[219, 223, 227, 708]|
|:large_blue_circle:|com/starrocks/load/loadv2/BulkLoadJob.java|0|1|00.00%|[228]|
|:large_blue_circle:|com/starrocks/load/EtlStatus.java|49|54|90.74%|[338, 339, 340, 341, 342]|

**Comment by padmejin** [comment]
run starrocks_admit_test

**Comment by padmejin** [comment]
@Mergifyio backport branch-2.3 branch-2.2 branch-2.1 branch-2.0
Gracias~ ¡Tenga un buen día!

**Pull request merged**